service/iot-service-samples/device-manager-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/DeviceManagerImportExportWithIdentitySample.java

```diff
@@ -50,10 +50,12 @@ public static void ExportDevices() throws IOException, IotHubException, Interrup
         ManagedIdentity identity = new ManagedIdentity();
         identity.setUserAssignedIdentity(userDefinedManagedIdentityResourceId);
 
-        JobProperties jobProperties = new JobProperties();
-        jobProperties.setOutputBlobContainerUri(blobContainerUri);
-        jobProperties.setStorageAuthenticationType(StorageAuthenticationType.IDENTITY);
-        jobProperties.setIdentity(identity);
+        JobProperties jobProperties = JobProperties.createForExportJob(
+                blobContainerUri,
+                false,
+                StorageAuthenticationType.IDENTITY,
+                identity
+        );
```

Comment:
Shouldn't this be on the end of line 57?

Reply:
Looks like intellij formatted it this way. Let me look elsewhere how it is.

Reply:
Fixing it.